spec/acceptance/hieradata/common.yaml

```diff
@@ -1,5 +1,13 @@
 ---
 apache::default_mods: false
+# This is normally set in the installer, but the bootstrap_rpm test needs it
```

Comment:
I didn't want to fully dive into it to fix the tests. This is already close enough now.